deviceenumerator_unix.cpp

```diff
@@ -111,7 +112,7 @@ QStringList DeviceEnumerator_unix::getUserFriendlyNames(const QStringList &devic
         QString output;
 
         // try to get label from first partititon
-        lsblk.start(QString("diskutil info %1s1").arg(device), QIODevice::ReadOnly);
+        lsblk.start("diskutil", {"info", device}, QIODevice::ReadOnly);
```

Comment:
previously you'd query info of e.g. `disk1s1`, now it's just `disk1`, is this correct? Also comment says "...from first partititon"

Reply:
Nice catch. Fixed.